manuscript.Rmd

```diff
@@ -68,55 +68,154 @@ set.seed(42)
 knitr::opts_chunk$set(cache.extra = knitr::rand_seed)
 ```
 
-Based on four basic principles of a more responsible research assessment in academic hiring and promotion processes [@schonbrodt2022responsible], @gartner2022responsible suggested a specific implementation of these principles to broaden the range of what constitutes research contributions. Gaertner et al. provide evaluation schemes for published manuscripts, published datasets, and software contributions. 
+# Introduction
+Based on four basic principles of a more responsible research assessment in academic hiring and promotion processes [@schonbrodt2022responsible], @gartner2022responsible suggested a specific implementation of these principles to broaden the range of what constitutes research contributions.
+Gaertner et al. provide evaluation schemes for published manuscripts, published datasets, and software contributions.
 We specifically endorse the increased emphasis on research software as recognizable and laudable scientific contribution.
-Why are research software contributions important scientific contributions? We would like to respond with a quote from the Science Code Manifesto (http://sciencecodemanifesto.org/):
-"Software is an essential research product, and the effort to produce, maintain, adapt, and 
-curate code must be recognized."
+Why are research software contributions important scientific contributions?
+We would like to respond with a quote from the Science Code Manifesto (http://sciencecodemanifesto.org/):
+"Software is an essential research product, and the effort to produce, maintain, adapt, and curate code must be recognized."
```

Comment:
Maybe switch the quotes. First, it is important, and because it is important and requires effort, it should be recognized.